cmd/fa-om-exporter/main.go

```diff
@@ -150,8 +150,9 @@ func metricsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+        uagent := r.Header.Get("User-Agent")
```

Comment:
Can we make the assumption that every http call will have a user-agent?

https://datatracker.ietf.org/doc/html/rfc2616#section-14.43

`User agents SHOULD include this field with requests.`

I would consider setting a default in case no user agent is provided

Reply:
I think it highly likely that this string will be provided. IF it isn't then we will return an empty string which should also be fine. It will just highlight that there is a client that is not well coded to the RFC.

Reply:
Looks good with the User-Agent concern that I had. 

See below on my testing

````
HEADERS:
        User-Agent: Pure_FA_OpenMetrics_exporter/1.0 ()
````